modular/gater/bucket_handler.go

```diff
@@ -69,7 +69,7 @@ func (g *GateModular) getBucketReadQuotaHandler(w http.ResponseWriter, r *http.R
 	}
 
 	bucketTraffic, err = g.baseApp.GfSpDB().GetBucketTraffic(
-		bucketInfo.Id.Uint64())
+		bucketInfo.Id.Uint64(), reqCtx.vars["year_month"])
```

Comment:
It is best not to directly operate the db in the gateway.

Reply:
fixed,  moved the logic to meta service